Makefile

```diff
@@ -369,7 +369,7 @@ format: \
 ## run gofumpt, goimports for all go files
 format/go:
 	find ./ -type d -name .git -prune -o -type f -regex '.*[^\.pb]\.go' -print | xargs gofumpt -w
-	find ./ -type d -name .git -prune -o -type f -regex '.*[^\.pb]\.go' -print | xargs goimports -w
+	find ./ -type d -name .git -prune -o -type f -regex '.*\.go' -print | xargs goimports -w
```

Comment:
should we ignore formatting generated files?

Reply:
yes, apis/grpc code was already goimports executed code, so if we don't run it will make many difference to this pr.

Reply:
and I think it would be okay to run gofumpt for it.